app/assets/stylesheets/application.css.scss

```diff
@@ -278,11 +278,17 @@ main {
     top: 9px;
     right: 2px;
     position: absolute;
-    border-radius: 5px;
     color: white;
+    border-radius: 30px;
     background-color: black;
     border: none;
 }
+.finalize {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-color, border, border-radius, color